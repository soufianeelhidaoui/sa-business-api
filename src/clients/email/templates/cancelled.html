<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
    <style>
    </style>
  </head>
  <body>
    <div style="max-width: 600px; margin: 0 auto;">
    {{> headBand }}
        <div style= 'font-size: 16px;
        font-style: normal;
        font-weight: 400;
        line-height: 24px;
        font-family: VW Head, Arial;
        margin-bottom: 32px
        '>
        <h2>
            <center>Votre commande a bien été annulée</center>
        </h2>
        <p>Bonjour {{ orderDetails.customer.firstName }},</p>
        <p>Votre commande a bien été annulée par votre partenaire. Il se tient à votre disposition pour toute demande d'information.
        </p>
        </div>
        <div class="wrapper" style="margin:0px auto;">
            {{> footer }}
        </div>
    </div>
  </body>
</html>
