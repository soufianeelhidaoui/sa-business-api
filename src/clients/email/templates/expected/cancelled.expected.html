<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
    <style>
    </style>
  </head>
  <body>
    <div style="max-width: 600px; margin: 0 auto;">
    <center>
      <img
        src="SKODA_API_URL/assets/shop/img/email/email-logo-skoda.png"
        alt="hero"
      />
    </center>
    <img
      src="SKODA_API_URL/assets/shop/img/email/email-skoda.jpg"
      alt="hero"
      style="width:100%"
    />
        <div style= 'font-size: 16px;
        font-style: normal;
        font-weight: 400;
        line-height: 24px;
        font-family: VW Head, Arial;
        margin-bottom: 32px
        '>
        <h2>
            <center>Votre commande a bien été annulée</center>
        </h2>
        <p>Bonjour Test,</p>
        <p>Votre commande a bien été annulée par votre partenaire. Il se tient à votre disposition pour toute demande d'information.
        </p>
        </div>
        <div class="wrapper" style="margin:0px auto;">
            <div>
                <hr style="color: #DFE4E8" />
                <!--[if mso | IE]><table align="center" border="0" cellpadding="0" cellspacing="0" class="" style="width:600px;" width="600" ><tr><td style="line-height:0px;font-size:0px;mso-line-height-rule:exactly;"><![endif]-->
                <!--[if mso | IE]><table align="center" border="0" cellpadding="0" cellspacing="0" class="wrapper-outlook" style="width:600px;" width="600" ><tr><td style="line-height:0px;font-size:0px;mso-line-height-rule:exactly;"><![endif]-->
                <div class="wrapper" style="margin:0px auto;">
                    <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" style="width:100%;">
                        <tbody>
                            <tr>
                                <td style="direction:ltr;font-size:0px;text-align:center;">
                                    <!--[if mso | IE]><table role="presentation" border="0" cellpadding="0" cellspacing="0"><tr><td class="" style="vertical-align:top;width:520px;" ><![endif]-->
                                    <div class="mj-column-per-100 mj-outlook-group-fix" style="font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;">
                                        <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="vertical-align:top;" width="100%">
                                            <tbody>
                                                <tr>
                                                    <td align="left" style="font-size:0px;padding:10px;word-break:break-word;">
                                                        <div style="font-family:SKODANext, Arial;font-size:12px;font-weight:300;line-height:24px;text-align:left;color:#878787;">
                                                            <p>SKODA, division de Volkswagen Group France responsable du traitement, traite les données à caractère personnel recueillies afin de vous permettre de recevoir, via l'adresse email que vous avez renseignée, les coordonnées et les références de votre Réparateur Agréé préféré sur la base légale de notre intérêt légitime, dans le cadre de la gestion de notre relation client.</p>
                                                            <p>Pour en savoir plus sur la gestion de vos données personnelles et pour exercer vos droits, veuillez consulter notre <a href="https://www.skoda.fr/informations/politique-de-confidentialite?_gl&#x3D;1*1mqpfd6*_gcl_au*OTk2NjU4OTQ1LjE3MTc1MTUwMjk.">Politiques de confidentialité</a> </p>
                                                        </div>
                                                    </td>
                                                </tr>
                                            </tbody>
                                        </table>
                                    </div>
                                </td>
                            </tr>
                        </tbody>
                    </table>
                </div>
              </div>
        </div>
    </div>
  </body>
</html>
